python_modules/dagster/dagster/core/definitions/composition.py

```diff
@@ -334,6 +334,21 @@ def __getitem__(self, idx):
             )
         )
 
+    def __getattr__(self, attr_called):
+        if attr_called == 'alias':
```

Comment:
I think better to just define `def alias(self, _):` , that way we don't swallow errors for invocations of other non-existent methods